python/ray/services.py

```diff
@@ -367,6 +371,51 @@ def check_version_info(redis_client):
             print(error_message)
 
 
+def start_credis(node_ip_address,
+                 port=None,
+                 redirect_output=False,
+                 cleanup=True):
+    """Start the credis global state store."""
+
+    stdout_file, stderr_file = new_log_files(
+        "credis_master", redirect_output)
+
+    master_port, _ = start_redis_instance(
+        node_ip_address=node_ip_address, port=port,
+        stdout_file=stdout_file, stderr_file=stderr_file,
+        cleanup=cleanup,
+        module="./core/src/credis/build/src/libmaster.so",
+        executable="./core/src/credis/redis/src/redis-server")
```

Comment:
Let's not hardcode these paths. It should be in a variable.

Also let's use absolute paths. This has caused issues in the past.